### PR TITLE
fix: retain scroll & expansion state of config list table

### PR DESCRIPTION
**Pull request by Prashanth-Nelli**
- [x] remove tooltips
- [x] style the popover it should look different from background ui & only one popover should be shown.
- [x] only clicking on more should open popover
- [x] if there is one item don't show popover
- [x] auto close popover after 5 seconds after opened.
- [x] improve performance of config list

### CODE REVIEW

**Comment by moshloop** [comment]
@Prashanth-Nelli can you also have a look at https://www.loom.com/share/40e8a3c1e37c4497bb0683d056e91444 - I think this was happening also before this PR

**Comment by moshloop** [comment]
@Prashanth-Nelli 

- We should still show at least 1 analysis, with horizontal ... for more - Same applies to tags, Show until it overflows and then 3 dots
- Use a popover, not a full modal 
- For changes - just count them

**Comment by moshloop** [comment]
@Prashanth-Nelli  - It is not clear that there are more items or that the tag is clickable - might need something like "3 more"

**Comment by Prashanth-Nelli** [comment]
Sure

File: src/components/Popover/Popover.tsx [hunk 1]
```diff
@@ -27,23 +29,27 @@ export default function Popover({
...
+        </button>
+      )}
+      {toggle}
       <div
         role="menu"
         aria-orientation="vertical"
         aria-labelledby="menu-button"
         className={clsx(
-          "flex flex-col origin-top-right absolute mt-5 w-96 z-50 divide-y divide-gray-100 rounded-md shadow-lg bg-white ring-1 ring-black ring-opacity-5 focus:outline-none capitalize",
+          "flex top-8 flex-col origin-top-right absolute w-96 z-50 divide-y divide-gray-100 rounded-md shadow-lg bg-white ring-1 ring-black ring-opacity-5 focus:outline-none capitalize",
```
Comment (moshloop) [inline src/components/Popover/Popover.tsx:52]:
```suggestion
          "flex top-8 flex-col origin-top-right absolute w-96 z-50 divide-y divide-gray-100 rounded-md shadow-lg bg-white ring-1 ring-black ring-opacity-5 focus:outline-none",
```

**Comment by moshloop** [comment]
@Prashanth-Nelli - the click target on the +4 more is too small - it should probably be the entire cell

**Pull request merged**